tools/oracle/run-differentials.sh

```
@@ -0,0 +1,111 @@
+report() { grep -E "live parity|parity:|value-identical|skipping|test result|diverges|stale" || true; }
+
+echo "== KMM differentials (gen+export, to-interpolation, merge, prune, validate) =="
+cargo test -p oxpinyin-kmm --test differential -- --nocapture 2>&1 | report
+echo "== segment: spseg / mergeseq =="
+cargo test -p oxpinyin-segment $feat --test spseg_mergeseq -- --nocapture 2>&1 | report
+echo "== segment: ngseg (needs the system bigram) =="
+cargo test -p oxpinyin-segment $feat --test differential -- --nocapture 2>&1 | report
+echo "== lambda: estimate_interpolation =="
+cargo test -p oxpinyin-lambda $feat --test differential -- --nocapture 2>&1 | report
+echo "== counter: gen_ngram =="
+cargo test -p oxpinyin-counter $feat --test differential -- --nocapture 2>&1 | report
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)